docs/learn/prompts.md

```diff
@@ -529,16 +580,83 @@ print(response.content)
 # > Here's a recommendation for a fantasy book: ...
 ```
 
+This feature allows you to use docstrings as prompt templates, which can be useful in certain scenarios where you want to keep the prompt close to the function or class definition.
+
 !!! warning
 
     Using docstrings as prompt templates can make your code less explicit and harder to maintain. It's generally recommended to use the `@prompt_template` decorator for clarity and separation of concerns. Enable this feature at your own risk.
 
 ## Best Practices
 
-- Provider Comparison: Use `BasePrompt` to easily test the same prompt across different providers to compare performance and output quality.
-- Prompt Versioning: Utilize the metadata decorator to keep track of different versions of your prompts as you refine them.
-- Dynamic Content: Leverage `@computed_field` for injecting dynamic content or API calls into your prompts.
-- Cached Properties: Use `@functools.cached_property` to cache frequently used properties that you only want to compute once.
-- Prompt Libraries: Build libraries of commonly used prompts that can be easily shared across projects or teams.
-
-Mastering `BasePrompt` is the first step towards building robust LLM applications with Mirascope that are flexible, reusable, and provider-agnostic.
+To make the most of Mirascope's prompt features, consider the following best practices:
+
+- **Provider Comparison**: Use `BasePrompt` to easily test the same prompt across different providers to compare performance and output quality.
+   ```python
+   prompt = MyPrompt(...)
+   openai_response = prompt.run(openai.call(...))
+   anthropic_response = prompt.run(anthropic.call(...))
+   ```
+
+- **Prompt Versioning**: Utilize the metadata decorator to keep track of different versions of your prompts as you refine them.
```

Comment:
I think we should move metadata like other things into a section where it makes more sense.

right now, metadata is generally contained either on `BasePrompt` or as part of a response to a call, so it seems like the best place to discuss metadata would be in the "Calls" documentation?